src/vanilla/shallow.ts

```diff
@@ -1,7 +1,6 @@
 export function shallow<T>(objA: T, objB: T) {
-  if (Object.is(objA, objB)) {
-    return true
-  }
```

Comment:
I kind of like this style. While we don't use it, eslint has a rule for this: https://eslint.org/docs/latest/rules/curly
I wonder if there's a rollup plugin or something to transpile it to remove `{}` at the build step.

Reply:
Ah good point. I'll take a look for plugins.

Reply:
I did not find anything on my first search. I'm sure there's a way to do this, but I'm not very fluent with Rollup plugins, so I'll close this PR for now because I don't intent to find a plugin. Thank you for understanding.